### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by crownk12** [RangeShifter/RangeshiftR-tutorials#19] [Q&A]
Hello, I tried to install RangeshiftR, but it is not working, I installed Rcpp, devtools, Rtools4.0, and Rdpack packages though. Error msg is just like this.

Error: Failed to install 'RangeShiftR' from GitHub:
  System command 'Rcmd.exe' failed, exit status: 1, stdout & stderr were printed

**Comment by JetteReeg** [comment]
Hi, 
could you please post which R version you are using?
Have you already googled the error message (e.g. https://stackoverflow.com/questions/63161905/problem-with-rcmd-exe-when-installing-packages) and checked that the PATH is set?

**Reply by crownk12** [reply]
> sessionInfo()

R version 4.1.2 (2021-11-01)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 22000)

Matrix products: default

locale:
[1] LC_COLLATE=Korean_Korea.949  LC_CTYPE=Korean_Korea.949    LC_MONETARY=Korean_Korea.949 LC_NUMERIC=C                
[5] LC_TIME=Korean_Korea.949    
system code page: 65001

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_4.1.2 tools_4.1.2

**Reply by crownk12** [reply]
I wonder if you can explain a little bit more about that...? How can I control PATH set?

**Reply by JetteReeg** [reply]
One additional question: 
Have you followed the tutorial for installing RangeShiftR? Have you not only installed the R packages Rcpp, devtools and Rdpack but also loaded it with [library()](https://rangeshifter.github.io/RangeshiftR-tutorials/installing.html#install-and-load-required-packages)? 
And have you tested the [toolchain](https://rangeshifter.github.io/RangeshiftR-tutorials/installing.html#compiler-toolchain) without error messages? At the end of this section you can also find a description how to set the PATH variable. But you could also have a look [here](https://www.hanss.info/sebastian/post/rtools-path/)

**Reply by crownk12** [reply]
If I add PATH, should I restart my computer?
